src/rules/color-named/README.md

```diff
@@ -105,3 +105,35 @@ a { color: $blue; }
 ```less
 a { color: @blue; }
 ```
+
+## Optional options
```

Comment:
Can this be updated to `Optional secondary options` please.

This was standardised in #1816